Update README.md (#2)
ducktype-org/.github: profile/README.md

Duckling is an in-development programming language.

> Docs and source code of compiler and related toolset not publcly avaiable yet.

Author: ```suggestion
Docs and source code of compiler and related toolset are not publicly available yet.
```

> We are planing to make first alpha version of both public this year (2025).

Author: ```suggestion
We are planning to make first alpha version of both public this year (2025).
```

> Docs and source code of compiler and related toolset are not publicly available yet.
> We are planning to make first alpha version of both public this year (2025).

Reviewer: ```suggestion
The documentation and source code of the compiler and related tools are not publicly available yet.
We are planning to publish their first alpha versions this year (2025).
```
